pkg/apis/v1beta1/taints.go

```diff
@@ -32,6 +34,15 @@ var (
 		Effect: v1.TaintEffectNoSchedule,
 		Value:  DisruptingNoScheduleTaintValue,
 	}
+
+	// DisruptionNonGracefulShutdown is used by the deprovisioning controller to forcefully
+	// shut down a node. This does not respect graceful termination of any pods on the node.
+	// https://kubernetes.io/docs/concepts/architecture/nodes/#non-graceful-node-shutdown
+	DisruptionNonGracefulShutdown = v1.Taint{
+		Effect: v1.TaintEffectNoExecute,
+		Key:    v1.TaintNodeOutOfService,
```

Comment:
nit: this is a super tiny style suggestion but consider going in Key, Effect, Value order to match the order that manifests typically look

Reply:
Sure thing, I used EKV because it's alphabetical.